docs/developer-docs/backend/candid/candid-howto.md

```diff
@@ -39,7 +39,7 @@ $ dfx canister call counter inc '(42)'
 
 To figure out how to create more complex Candid arguments, please refer to the [Candid Reference](/references/candid-ref.md). And for Candid arguments too long to fit the command line, please use the `--argument-file` flag of [`dfx canister call`](/references/cli-reference/dfx-canister.md#dfx-canister-call).
 
-You can also omit the arguments and let `dfx` generate a random value that matches the method type. For example:
+You can also omit the arguments and let the IC SDK generate a random value that matches the method type. For example:
```

Comment:
This should be reverted; dfx is what's generating the random value

Reply:
tbh, i am trying to lower the importance of `dfx` and so i raised the the level of abstraction to `IC SDK` (since it contains `dfx`). Is that wrong? wdyt?